editor/src/core/workers/parser-printer/parser-printer.spec.ts

```diff
@@ -4566,9 +4570,8 @@ export var App = props => {
 
     const position = consumer.getOriginalPosition(transpiledLine, transpiledCharacter)
 
-    // TODO BALAZS we should test that the code is in col 9, but I just couldn't make it work :(
-    // expect(position).toEqual(expect.objectContaining({ line: 22, column: 9 }))
-    expect(position).toEqual(expect.objectContaining({ line: 16 }))
+    // FIXME This should be line 16 char 9 but I cannot get this to work
+    expect(position).toEqual(expect.objectContaining({ line: 17, column: 10 }))
```

Comment:
btw what is wrong with this test? :D

Reply:
Honestly I have no idea and gave up at this point trying to fix it. I tested inserting an error in that position in the actual editor and the error overlay mapped to the correct line and character so :shrug: